sunpy/image/slit.py

```diff
@@ -0,0 +1,129 @@
+import sunpy
+import sunpy.map
+import sunpy.cm
+import sunpy.wcs
+
+import numpy as np
+import astropy.wcs
+import astropy.io.fits as fits
+import astropy.units as u
+
+import scipy.ndimage as snd
+import matplotlib.pyplot as plt
+from glob import glob
+
+
+
+def slit(in_files, xy1, xy2):
+    """
+    Returns an array with intensity along the slit on the y axis and time 
+    along x.
+    
+    Parameters    
+    ----------
+    in_files : `sunpy.map.MapCube`
+        A mapcube of the images you want to perform the slit analysis on. 
+        Usually with x and y as space, and z as time.
+    xy1 : List of x and y coordinates in `[x1, y1]`
```

Comment:
I would make it take two Quantity objects, i.e. x,y pairs. Like this: http://docs.sunpy.org/en/stable/api/sunpy.map.GenericMap.html#sunpy.map.GenericMap.submap